src/renderer/library/components/dialog/publicationInfos/catalogControls.tsx

```diff
@@ -53,26 +55,26 @@ export class CatalogControls extends React.Component<IProps, undefined> {
         return (
             <>
                 <button onClick={this.handleRead} className={stylesButtons.button_primary}>
+                    <SVG svg={ReadBook} ariaHidden />
                     {__("catalog.readBook")}
                 </button>
-                <DeletePublicationConfirm
-                    trigger={(
-                        <button className={stylesButtons.button_transparency}>
-                            <SVG svg={DeleteIcon} ariaHidden />
-                            {__("catalog.deleteBook")}
-                        </button>
+                <button onClick={this.deletePublication} className={stylesButtons.button_transparency}>
```

Comment:
Why delete here the new radix alert component :  `DeletePublicationConfirm` ?

Reply:
Yes, sorry it was a mistake, I hadn't realised that.